src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -194,10 +194,17 @@ public partial class SendPacketsElement
         public SendPacketsElement(string filepath) { }
         public SendPacketsElement(string filepath, int offset, int count) { }
         public SendPacketsElement(string filepath, int offset, int count, bool endOfPacket) { }
+        public SendPacketsElement(string filepath, long offset, int count) { }
+        public SendPacketsElement(string filepath, long offset, int count, bool endOfPacket) { }
+        public SendPacketsElement(System.IO.FileStream fileStream) { }
+        public SendPacketsElement(System.IO.FileStream fileStream, long offset, int count) { }
+        public SendPacketsElement(System.IO.FileStream fileStream, long offset, int count, bool endOfPacket) { }
         public byte[] Buffer { get { throw null; } }
         public int Count { get { throw null; } }
+        public long LongOffset { get { throw null; } }
```

Comment:
The API was approved as `OffsetLong`:
https://github.com/dotnet/corefx/issues/25354#issuecomment-352854380